### ISSUE TITLE
A little suggestion

### ISSUE THREAD

**Issue by ilovexy2tom** [mytooyo/board_datetime_picker#57]
Hello, your library looks great. I have used it in my project, but there is actually a small problem. Users don't have to make a selection every time. They can cancel and keep the last selected time, but I can't add it to the default title for now.

I hope to add a feature that allows users to abandon this selection.

Now my experience is: clicking outside the screen will return null, which is equivalent to resetting.

Logically:

Click reset: return null
Click confirm: return date
Click outside the screen: return nothing (user still keeps the previously selected date)

**Comment by ilovexy2tom** [comment]
![2024-12-27 12 14 00](https://github.com/user-attachments/assets/16ebd3cd-094f-4a12-a5fa-81d84e5007af)

Similar to how I can add a reset or cancel here

**Comment by ilovexy2tom** [comment]
> ![2024-12-27 12 14 00](https://private-user-images.githubusercontent.com/22984252/398844754-16ebd3cd-094f-4a12-a5fa-81d84e5007af.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyODI1MjQsIm5iZiI6MTczNTI4MjIyNCwicGF0aCI6Ii8yMjk4NDI1Mi8zOTg4NDQ3NTQtMTZlYmQzY2QtMDk0Zi00YTEyLWE1ZmEtODFkODRlNTAwN2FmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDA2NTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1OTg5ZjM2OTkzNDZlZjk1NTc4NjVlOGU2NGQzM2I1YWNmZDA2ZTRhYTg5NDU4OWIzNWUyMGIxMDRkM2IxNWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PSKdocFJ23pvCCwRKmglNsQSz2EJUditiLXLLnblUcs)
> 
> Similar to how I can add a reset or cancel here

I read the source code carefully and found that single selection supports TopWidget customization, but multiple selection does not support TopWidget and Controller. I hope that subsequent versions can add

**Comment by mytooyo** [comment]
@ilovexy2tom 
You’re right; it might not have supported multiple selections. I will fix it along with bug #58. Please wait a moment.

**Comment by mytooyo** [comment]
@ilovexy2tom 
I have made it possible to use TopWidget and Controller even with multiple selection.
Since I thought it would be useful to have only the reset function, I added the useResetButton option.
This should be convenient if you want to enable resetting.

```dart
options: BoardDateTimeOptions(
  ...,
  useResetButton: true,
)
```
